db/migrate_cards/20150618122941_codename_for_homepage.rb

```diff
@@ -0,0 +1,7 @@
+# -*- encoding : utf-8 -*-
+
+class CodenameForHomepage < Card::Migration
+  def up
+    Card['home'].update_attributes! :codename=>:home
+  end
+end
```

Comment:
"home" is already in use as a codename for the card used to set the homepage:

http://wikirate.org/:home

...so I think this would break.